### PR TITLE
Completed typescript-local-storage - Nader Motamedi

### PR DESCRIPTION
**Pull request by nmotamedi**
local-storage

### CODE REVIEW

File: typescript-local-storage/typescript-local-storage-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- How do you store data in `localStorage`?
+
+- How do you retrieve data from `localStorage`?
+
+- What data type can `localStorage` save in the browser?
+
+- When does the `'beforeunload'` event fire on the `window` object?
+
```
Comment (shawnkost) [inline typescript-local-storage/typescript-local-storage-quiz-notes.md:14]: Missing the quiz questions here!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just missing the quiz questions

File: typescript-local-storage/typescript-local-storage-quiz-notes.md (hunk 1 above)
Comment (nmotamedi) [reply typescript-local-storage/typescript-local-storage-quiz-notes.md]: Just noticed and updated!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**